app/helpers/solr_helper.rb

```diff
@@ -4,7 +4,7 @@ def solr_name(field_name, *opts)
     if Module.const_defined?(:Solrizer)
       ::Solrizer.solr_name(field_name, *opts)
     else
-      ::ActiveFedora.index_field_mapper.solr_name(field_name)
+      ::ActiveFedora.index_field_mapper.solr_name(field_name, *opts)
```

Comment:
For reference: https://github.com/samvera/active_fedora/blob/99ac8d1e24bf0ee1673187ae6c0f3456ae9c04c0/lib/active_fedora/indexing/field_mapper.rb#L29